products/gke.md

```diff
@@ -3,6 +3,12 @@ title: Google Kubernetes Engine
 category: service
 changelogTemplate: https://cloud.google.com/kubernetes-engine/docs/release-notes-nochannel
 releases:
+-   releaseCycle: "1.25"
+    eol: 2024-03-31
+    support: 2024-01-31
+    releaseDate: 2022-08-23
```

Comment:
Our page currently is only tracking `nochannel` releases. To properly track other channels, we need to duplicate all of the releases, which will make it quite complicated. Just considering nochannel:

1. 1.25 was released on nochannel on 14th December as per https://cloud.google.com/kubernetes-engine/docs/release-notes-nochannel#December_14_2022 (Our gke.json agrees: https://github.com/endoflife-date/release-data/blob/main/releases/gke.json)
2. EOL Dates for all channels are the same, so we use 2024-02-29 as mentioned on https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels
3. Support Dates are EOL - 2months, keeping with standard

```suggestion
-   releaseCycle: "1.25"
    eol: 2024-02-29
    support: 2023-12-29
    releaseDate: 2022-12-14
```